src/implementation.md

```diff
@@ -67,6 +74,12 @@ To support the light client functionality we will implement the support of the f
 - /dot/sync/1
 - /dot/light/2
 
-We will use hash_map with serialization/deserialization to hold the blockchain state.
+## Database
+
+We will use a hash map to store the blockchain state.  This is an in-memory serialization/deserialization data structure, functioning as a de facto database.
```

Comment:
I would add more details: "to store current peers and block header information for some number of blocks in history"

Reply:
:heavy_check_mark: